### PR TITLE
Improve customer basic info section

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Improved responsivity of basic information block in customer zone. Extended wording on guest conversion.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | See that responsivity is better.
| Fixed ticket?     | 
| Related PRs       | 
| Sponsor company   | 

basicdesktop
basicmobile

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/personal_information.html.twig [hunk 1]
```diff
@@ -161,27 +161,27 @@
...
+      <div class="col-sm-4 text-sm-right font-weight-bold">
+          {{ 'Customer type'|trans({}, 'Admin.Orderscustomers.Feature') }}
+      </div>
+      <div class="col-sm-8">
+        {% if customerInformation.personalInformation.guest %}
+            <p class="mb-0">{{ 'Guest customer'|trans({}, 'Admin.Orderscustomers.Feature') }}</p>
+            {% if customerInformation.generalInformation.customerBySameEmailExists %}
+              <p class="text-muted">{{ 'You can\'t transform this guest to a registered customer, because a registered customer with this email address already exists. '|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
+            {% else %}
+              <p class="text-muted">{{ 'You can transform this guest to a registered customer. The customer will receive an email with a link to set his password.'|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
```
Comment (kpodemski) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/personal_information.html.twig]:
```suggestion
              <p class="text-muted">{{ 'You can transform this guest to a registered customer. The customer will receive an email with a link to set the password.'|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
```

probably better this way, wording has to be check with @l-delin

Comment (l-delin) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/personal_information.html.twig]:
```suggestion
              <p class="text-muted">{{ 'You can transform this guest to a registered customer. The customer will receive an email with a link to set a password.'|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
```

Comment (l-delin) [reply src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/personal_information.html.twig]: I think "a" would be better here, as it's more neutral :-)

**Comment by Hlavtox** [comment]
@l-delin @kpodemski Wording applied :-)

**Review by nicosomb — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @Hlavtox 

Thank you for your PR, I tested it and, as you can see, it seems to works as expected :

image

**Pull request merged**